packages/cw3/src/msg.rs

```diff
@@ -0,0 +1,64 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use std::fmt;
+
+use cosmwasm_std::{CosmosMsg, Empty};
+use cw0::Expiration;
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
+#[serde(rename_all = "snake_case")]
+pub enum Cw3HandleMsg<T = Empty>
+where
+    T: Clone + fmt::Debug + PartialEq + JsonSchema,
+{
+    Propose {
+        title: String,
+        description: String,
+        msgs: Vec<CosmosMsg<T>>,
+        expires: Option<Expiration>,
+    },
+    Vote {
+        proposal_id: u64,
+        vote: Vote,
+    },
+    Execute {
+        proposal_id: u64,
+    },
+    Close {
+        proposal_id: u64,
+    },
+}
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
+#[serde(rename_all = "lowercase")]
+pub enum Vote {
+    YES,
+    NO,
+    ABSTAIN,
+    VETO,
+}
```

Comment:
In the rest of the cosmwasm codebase, we use `PascalCase` for enum cases (Yes, No, …)

Reply:
Will update. Thanks